include/alpaka/mem/view/ViewStdArray.hpp

```diff
@@ -29,7 +29,7 @@ namespace alpaka::trait
     {
         ALPAKA_FN_HOST static auto getDev(std::array<TElem, Tsize> const& /* view */) -> DevCpu
         {
-            return getDevByIdx<PltfCpu>(0u);
+            return getDevByIdx(PltfCpu{}, 0u);
         }
```

Comment:
This is a hack we can do internally, because we know that the CPU platform does not contain any data. But it generally does not apply.

Reply:
Shouldn't this be a C++ comment?

Reply:
Possibily.